libs/platform/ebpf_tracelog.c

```diff
@@ -0,0 +1,1059 @@
+        _EBPF_LOG_MESSAGE_GUID_GUID_STRING(trace_level, KEYWORD_CORE, message, string, guid1, guid2);                \
+        break;                                                                                                       \
+    CASE_LINK:                                                                                                       \
+        _EBPF_LOG_MESSAGE_GUID_GUID_STRING(trace_level, KEYWORD_LINK, message, string, guid1, guid2);                \
+        break;                                                                                                       \
+    CASE_MAP:                                                                                                        \
+        _EBPF_LOG_MESSAGE_GUID_GUID_STRING(trace_level, KEYWORD_MAP, message, string, guid1, guid2);                 \
+        break;                                                                                                       \
+    CASE_PROGRAM:                                                                                                    \
+        _EBPF_LOG_MESSAGE_GUID_GUID_STRING(trace_level, KEYWORD_PROGRAM, message, string, guid1, guid2);             \
+        break;                                                                                                       \
+    CASE_API:                                                                                                        \
+        _EBPF_LOG_MESSAGE_GUID_GUID_STRING(trace_level, KEYWORD_API, message, string, guid1, guid2);                 \
+        break;                                                                                                       \
+    CASE_PRINTK:                                                                                                     \
+        _EBPF_LOG_MESSAGE_GUID_GUID_STRING(trace_level, KEYWORD_PRINTK, message, string, guid1, guid2);              \
+        break;                                                                                                       \
+    CASE_NATIVE:                                                                                                     \
+        _EBPF_LOG_MESSAGE_GUID_GUID_STRING(trace_level, KEYWORD_NATIVE, message, string, guid1, guid2);              \
+        break;                                                                                                       \
+    default:                                                                                                         \
+        break;                                                                                                       \
+    }
+__declspec(noinline) void ebpf_log_message_guid_guid_string(
+    ebpf_tracelog_level_t trace_level,
+    ebpf_tracelog_keyword_t keyword,
+    _In_z_ const char* message,
+    _In_z_ const char* string,
+    _In_ const GUID guid1,
+    _In_ const GUID guid2)
```

Comment:
```suggestion
    GUID guid1,
    GUID guid2)
```

Reply:
Changed the existing implementation to use pointers instead.